src/resources/MachineLearning/SmartSnippetsConfiguration/SmartSnippetsConfigurationInterfaces.ts

```diff
@@ -40,6 +46,10 @@ export interface SmartSnippetsDocumentGroupPreviewParams {
      * The sources to consider.
      */
     sources: string[];
+    /**
+     * An array of filtering conditions.
+     */
+    filterConditions: FilterConditions[];
 }
 
 export interface SmartSnippetsDocumentGroupPreview {
```

Comment:
Does this need to be updated as well? The api now returns an additional field `numberOfDocumentsMatchingConditions` with the following documentation (still pending doc approval) : `The number of documents that match the conditions.`

Also, the documentation for `numberOfDocumentsWithPermanentId` will be updated to `The number of documents that match the conditions and have a permanent id.` once we get doc approval.

Reply:
So I had actually made these changes, but now I realise I put them in the `master` branch, which I was allowed to push from my machine?? Anyway, fixed now.